### PR TITLE
Added new map "Red Zone"

### LINKED ISSUE #12596: Add "Red Zone" to official map pool.
This is a request to add "Red Zone" mission to the custom mission tab of the map pool. It was a part of last year's Map Design Contest and winner.

http://resource.openra.net/maps/15952/

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/OpenRA/issues/12596.

### CODE REVIEW

File: mods/ra/maps/red-zone/weapons.yaml [hunk 1]
```diff
@@ -0,0 +1,153 @@
+90mmAA:
+	Inherits: 90mm
+	InvalidTargets: Ground, Water
```
Comment (reaperrr) [inline mods/ra/maps/red-zone/weapons.yaml]: `Ground` and `Water` no longer stand for both terrain and actors, take a look at bleed's weapon definitions to update this accordingly (same for warheads).

**Comment by Mailaender** [comment]
Pinging @Quetip

**Comment by Quetip** [comment]
I need to dust off my RA coding skills and have a look a this.

File: mods/ra/maps/red-zone/weapons.yaml (hunk 1 above)
Comment (Mailaender) [reply mods/ra/maps/red-zone/weapons.yaml]: Changed it and removed some duplication from the inherited definitions as well.

**Comment by Mailaender** [comment]
Removed the BOM which Travis didn't like.

**Comment by Quetip** [comment]
acknowledging this.

**Comment by Mailaender** [comment]
Glad someone takes over as I don't seem to find the time.

**Pull request closed without merging**